api/api_full.go

```diff
@@ -153,6 +153,11 @@ type FullNode interface {
 	PaychVoucherSubmit(context.Context, address.Address, *paych.SignedVoucher) (cid.Cid, error)
 }
 
+type FileRef struct {
+	Path  string
+	IsCAR bool
```

Comment:
Let's take this one step further and just have a stringly typed 'ImportType' field, which we can set to car. I imagine there will be other potentials here in the future